packages/account/src/Components/poi-poa-docs-submitted/poi-poa-docs-submitted.tsx

```diff
@@ -41,23 +44,24 @@ const PoiPoaDocsSubmitted = ({
     };
 
     const getDescription = () => {
-        const { manual_status, poi_verified_for_maltainvest, poi_verified_for_bvi_labuan_vanuatu, poa_pending } =
-            getAuthenticationStatusInfo(account_status);
+        const { maltainvest, bvi_labuan_vanuatu } = poi;
         const is_maltainvest_selected = jurisdiction_selected_shortcode === Jurisdiction.MALTA_INVEST;
-        if (
-            (is_maltainvest_selected && poi_verified_for_maltainvest && poa_pending) ||
-            (!is_maltainvest_selected && poi_verified_for_bvi_labuan_vanuatu && poa_pending) ||
-            manual_status === 'pending'
-        ) {
-            return localize('We’ll review your documents and notify you of its status within 1 - 3 working days.');
+        const check_for_mf = is_maltainvest_selected && maltainvest.verified && poa.status === STATUSES.PENDING;
+        const check_for_cr = !is_maltainvest_selected && bvi_labuan_vanuatu.verified && poa.status === STATUSES.PENDING;
+        if (check_for_mf || check_for_cr || identity_status.manual === STATUSES.PENDING) {
```

Comment:
a doubt, is it ok to keep this decision making logic here? or should this be a single flag coming from hook/backend?